packages/riipen-ui/src/components/Paper.jsx

```diff
@@ -35,7 +35,6 @@ const Paper = props => {
       <style jsx>{`
         .root {
           background-color: ${theme.palette.common.white};
-          display: inline-block;
```

Comment:
This is the real fix here. Now this component will render with default `display` depending on its Component type.

Reply:
Oh yeah that's perfect